helium/config.json

```diff
@@ -28,33 +28,195 @@
      "//fonts.googleapis.com/css?family=Open+Sans:400italic,400,700"
   ],
   "extjs" : [],
-  "defaults": {
```

Comment:
We should leave the defaults in for backwards compatibility until we are sure all clients are using the new theme builder